corehq/util/__init__.py

```diff
@@ -1,5 +1,7 @@
 import re
 
+from .view_utils import reverse
```

Comment:
@benrudolph I think he's just using the convention of putting the public API of a module into the `__init__.py` file.